### ISSUE TITLE
Train_loss and Validation_loss didn't change

### ISSUE THREAD

**Issue by suyanzhou626** [shekkizh/FCN.tensorflow#96]
When I train this FCN, the train_loss and validation loss always about 3. like follow picture.
![image](https://user-images.githubusercontent.com/37359271/38907169-12f6a014-42ed-11e8-9909-5990c896c273.png)

**Comment by shekkizh** [comment]
You need to train a lot longer - at least one epoch
